model/vm_host.rb

```diff
@@ -243,16 +244,83 @@ def reset
     Hosting::Apis.reset_server(self)
   end
 
+  def check_storage_smartctl(ssh_session, devices)
+    devices.map do |device_name|
+      result = ssh_session.exec!("sudo smartctl -j -H /dev/#{device_name} | jq .smart_status.passed").strip == "true"
+      result.tap do |passed|
+        Clog.emit("Device #{device_name} failed smartctl check.") unless passed
+      end
```

Comment:
```suggestion
      passed = ssh_session.exec!("sudo smartctl -j -H /dev/#{device_name} | jq .smart_status.passed").strip == "true"
      Clog.emit("Device #{device_name} failed smartctl check.") unless passed
      passed
```

`tap` generally makes sense if you are using it during a method chain.  If you already have a local variable, it's simpler to just use it.  Same for other methods in this file.

Reply:
got it thanks. fixed it